reference/forms/types/timezone.rst

```diff
@@ -49,12 +49,12 @@ you should just use the ``choice`` type directly.
 Overridden Options
 ------------------
 
-choice_list
+choices
 ~~~~~~~~~~~
```

Comment:
A very minor comment: in the documentation we always make the below underlining characters match the length of the above heading:

    choices
    ~~~~~~~